src/runtime/disco/builtin.cc

```diff
@@ -84,6 +84,8 @@ void AllReduce(NDArray send, ReduceKind reduce_kind, NDArray recv) {
   GetCCLFunc("allreduce")(send, static_cast<int>(reduce_kind), recv);
 }
 
+void AllGathere(NDArray send, NDArray recv) { GetCCLFunc("allgather")(send, recv); }
```

Comment:
Typo: This should be `AllGather` instead of `Allgathere`.

Reply:
Thanks for catching this.